tests/zpill.py

```diff
@@ -105,9 +106,46 @@ def serialize(obj):
     raise TypeError("Type not serializable")
 
 
+# We cannot serialize datetime by default in json dumps
+# use this method to override default behavior
+def datetime_converter(obj):
+    if isinstance(obj, datetime):
+        return obj.isoformat()
+
+
+def save_response(self, service, operation, response_data,
```

Comment:
let's use a subclass, and override the method, that will be much smaller diff afaics

Reply:
ie you can do the modification on the response and then just call super() to the base for its logic.

Reply:
Will refactor to use super shortly. On second look, I am seeing that what I did here is duplicitous because all I am doing is changing the response data, thanks for pointing this out!